src/drivers/display/fb/lv_linux_fbdev.c

```diff
@@ -312,16 +312,20 @@ static void flush_cb(lv_display_t * disp, const lv_area_t * area, uint8_t * colo
         return;
     }
 
-    uint32_t color_pos = (area->x1 + dsc->vinfo.xoffset) * px_size + area->y1 * dsc->finfo.line_length;
-    uint32_t fb_pos = color_pos + dsc->vinfo.yoffset * dsc->finfo.line_length;
+    uint32_t color_pos =
+        area->x1 * px_size +
+        area->y1 * disp->hor_res * px_size;
```

Comment:
Please move it into the `if(LV_LINUX_FBDEV_RENDER_MODE == LV_DISPLAY_RENDER_MODE_DIRECT)` branch to indicate that it's used only there.

Reply:
Done.